code/game/mecha/equipment/tools/sec_tools.dm

```diff
@@ -108,25 +97,33 @@
 		return 1
 	return
 
-/obj/item/mecha_parts/mecha_equipment/tool/jail/proc/go_out(var/list/L)
-	var/mob/living/ejected = L["mob"]
+/obj/item/mecha_parts/mecha_equipment/tool/jail/proc/go_out(var/L)
```

Comment:
Changes the argument name to be something relevant to what it actually is now, like `cell_name`.